### DISCUSSION TITLE
Limit the PNP search to a specific AAD Group

### DISCUSSION THREAD

**Discussion by DanRuz54** [microsoft-search/pnp-modern-search#2642] [Q&A]
Is it possible to limit the PNP search to a specific AAD Group  instead of showing people in this Group?

I would like to  create [a useful People Search](https://microsoft-search.github.io/pnp-modern-search/scenarios/Create-a-useful-People-Search/) that will only show users from the AAD group.

I've tried these steps, but none of them work or I'm doing something wrong.

- https://sharepoint.stackexchange.com/questions/289833/sharepoint-search-query-display-people-from-a-specific-ad-group
- https://www.rlvision.com/blog/how-to-create-searchable-user-profile-properties-in-sharepoint-online/
- https://answers.microsoft.com/en-us/msoffice/forum/all/sp-online-and-memberof-attribute/4bdcbcad-dc35-4e0d-910f-51f7f0e7ff63

Thank you for any suggestions or help. 

Dan

**Comment by FranckyC** [comment]
HI @DanRuz54, I don't think you can achieve this using SharePoint/Microsoft Search. Using PnP Modern Search, the best you could do is to create a [custom data source](https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_data_sources/) using the [Graph API](https://learn.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0&tabs=http).
